packages/ui/src/working-groups/hooks/useOpeningsPagination.ts

```diff
@@ -27,7 +27,7 @@ export const useOpeningsPagination = ({ groupId, type, page = 1 }: UseOpeningsPa
     limit: OPENINGS_PER_PAGE,
     offset: (page - 1) * OPENINGS_PER_PAGE,
     where,
-    order: [WorkingGroupOpeningOrderByInput.CreatedAtDesc, WorkingGroupOpeningOrderByInput.RuntimeIdDesc],
+    order: [WorkingGroupOpeningOrderByInput.CreatedAtDesc],
```

Comment:
If I understand correctly this https://github.com/Joystream/pioneer/issues/2513#issuecomment-1575299539, the issue is that the openings are ordered by ascending creation date. So this line (`order: [WorkingGroupOpeningOrderByInput.CreatedAtDesc, WorkingGroupOpeningOrderByInput.RuntimeIdDesc]` looks fine to me, but the problem is still not solved on [the preview branch `/working-groups/openings`](https://dao-git-fork-chrlschwb-fix-2513-sortinglist-joystream.vercel.app/#/working-groups/openings):
![image](https://github.com/Joystream/pioneer/assets/6571453/a071e806-4655-4a6e-b508-ba5cf359b29c)

So I'm guessing that maybe this `useOpeningsPagination` is used to query the openings on a different page. E.g on the WG past openings page the order is correct even [in production](https://pioneerapp.xyz/#/working-groups/builders).

In conclusion I think this change should be reversed and either the hook used to query `/working-groups/openings` should be fixed or an even better solution would be to reuse the same hook to query openings everywhere.

Also please get reviews from BWG workers before setting `jsg-code-review`.